### PR TITLE
Hive 10689

### PR DESCRIPTION
**Pull request by thejasmn**

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HivePrivilegeObjectUtils.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * Utility functions for working with HivePrivilegeObject
+ */
+@LimitedPrivate(value = { "Apache Argus (incubating)" })
+@Evolving
+public class HivePrivilegeObjectUtils {
+
+  /**
+   * Convert list of dbnames into list of HivePrivilegeObject
+   * @param dbList
+   * @return
```
Comment (swarnim87) [inline ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HivePrivilegeObjectUtils.java]: Finish javadoc

**Comment by swarnim87** [comment]
+1

**Comment by vaibhavgumashta** [comment]
Looks good, +1.

**Comment by belugabehr** [comment]
Issue has been resolved through manual patch process:

https://issues.apache.org/jira/browse/HIVE-10689

**Pull request closed without merging**